magicctapipe/scripts/lst1_magic/semi_automatic_scripts/coincident_events.py

```diff
@@ -0,0 +1,374 @@
+                        ):
+                            os.mkdir(
+                                f"{coincidence_DL1_dir}/DL1Coincident/{p}/NSB{nsb}/{lstObsDir}"
+                            )
+                        if not os.path.exists(
+                            f"{coincidence_DL1_dir}/DL1Coincident/{p}/NSB{nsb}/{lstObsDir}/logs"
+                        ):
+                            os.mkdir(
+                                f"{coincidence_DL1_dir}/DL1Coincident/{p}/NSB{nsb}/{lstObsDir}/logs"
+                            )
+
+                        outputdir = f"{coincidence_DL1_dir}/DL1Coincident/{p}/NSB{nsb}/{lstObsDir}"
+                        list_of_subruns = np.sort(
+                            glob.glob(f"{inputdir}/dl1*Run*{i[1]}*.*.h5")
+                        )
+                        if os.path.exists(f"{outputdir}/logs/list_LST"):
+                            with open(
+                                f"{outputdir}/logs/list_LST", "a"
+                            ) as LSTdataPathFile:
+                                for subrun in list_of_subruns:
+                                    LSTdataPathFile.write(
+                                        f"{subrun}\n"
+                                    )  # If this files already exists, simply append the new information
+                        else:
+                            with open(
+                                f"{outputdir}/logs/list_LST.txt", "w"
+                            ) as f:  # If the file list_LST.txt does not exist, it will be created here
+                                for subrun in list_of_subruns:
+                                    f.write(f"{subrun}\n")
+
+                        if not os.path.exists(f"{outputdir}/logs/list_LST.txt"):
+                            continue
+                        process_size = (
+                            len(
+                                np.genfromtxt(
+                                    f"{outputdir}/logs/list_LST.txt", dtype="str"
+                                )
+                            )
+                            - 1
+                        )
+
+                        if process_size < 0:
+                            continue
+                        lines = [
+                            "#!/bin/sh\n\n",
+                            "#SBATCH -p short\n",
+                            f'#SBATCH -J {target_dir.split("/")[-2:][1]}_coincidence_{nsb}\n',
+                            f"#SBATCH --array=0-{process_size}\n",
+                            "#SBATCH --mem=30g\n",
+                            "#SBATCH -N 1\n\n",
+                            "ulimit -l unlimited\n",
+                            "ulimit -s unlimited\n",
+                            "ulimit -a\n\n",
+                            f"export INM={MAGIC_DL1_dir}/Merged/Merged_{str(Y_M).zfill(4)}_{str(M_M).zfill(2)}_{str(D_M).zfill(2)}\n",
```

Comment:
I think instead of Merged_{str(Y_M).zfill(4)}_{str(M_M).zfill(2)}_{str(D_M).zfill(2)} you can use d, right?